python/cuml/decomposition/tsvd.pyx

```diff
@@ -273,7 +272,7 @@ class TruncatedSVD(Base):
        X : array-like (device or host) shape = (n_samples, n_features)
            Dense matrix (floats or doubles) of shape (n_samples, n_features).
            Acceptable formats: cuDF DataFrame, NumPy ndarray, Numba device
-           ndarray, cuda array interface compliant array like CuPy
+           ndarray, rmm.array interface compliant array like CuPy
```

Comment:
Failed find and replace 😄

Reply:
this is referring to the `__cuda_array_interface__` and not RMM ;)